pkg/injector/patch.go

```diff
@@ -54,22 +54,29 @@ func (wh *mutatingWebhook) createPatch(pod *corev1.Pod, req *admissionv1.Admissi
 	// Create volume for envoy TLS secret
 	pod.Spec.Volumes = append(pod.Spec.Volumes, getVolumeSpec(envoyBootstrapConfigName)...)
 
-	// Build outbound port exclusion list
-	podOutboundPortExclusionList, _ := wh.getPortExclusionListForPod(pod, namespace, outboundPortExclusionListAnnotation)
-	globalOutboundPortExclusionList := wh.configurator.GetOutboundPortExclusionList()
-	outboundPortExclusionList := mergePortExclusionLists(podOutboundPortExclusionList, globalOutboundPortExclusionList)
-
-	// Build inbound port exclusion list
-	podInboundPortExclusionList, _ := wh.getPortExclusionListForPod(pod, namespace, inboundPortExclusionListAnnotation)
-	globalInboundPortExclusionList := wh.configurator.GetInboundPortExclusionList()
-	inboundPortExclusionList := mergePortExclusionLists(podInboundPortExclusionList, globalInboundPortExclusionList)
-
-	// Add the Init Container
-	initContainer := getInitContainerSpec(constants.InitContainerName, wh.configurator, wh.configurator.GetOutboundIPRangeExclusionList(), outboundPortExclusionList, inboundPortExclusionList, wh.configurator.IsPrivilegedInitContainer())
-	pod.Spec.InitContainers = append(pod.Spec.InitContainers, initContainer)
+	// On Windows we cannot use init containers to program HNS because it requires elevated privillages
```

Comment:
```suggestion
	// On Windows we cannot use init containers to program HNS because it requires elevated privileges
```

Reply:
Thanks for catching this, fixed in a8f1df1ea78e0932a6f67f3c51cdb6c6bbf8eb8f